Fix matching end of variable filter expression.

adamalton opened the pull request:
Fixes #4.

Now correctly ends the filter expression if the closing curly braces are encountered but with no space before them.

adamalton committed:
Fix matching end of variable filter expression.

Now correctly ends the filter expression if the closing curly braces are encountered but with no space before them.

willstott101 commented:
Thanks for digging into the regexes for this. Would you mind rebasing/merging and adding a test? They're pretty easy to write but can make these fiddly things way easier to keep working.

Thanks!

willstott101 commented:
(all the tests are in the **syntax_test_django.html.djt** file)

adamalton committed:
Add test for variable filters without surrounding spaces.

adamalton committed:
Add changes to CSS and XML variants for fix to filter expression without surrounding spaces.

adamalton committed:
Tweak README for clarity.

adamalton commented:
I'd glanced briefly at the `syntax_test_django.html.djt` file before, but at first look was totally baffled by it! Having now educated myself by reading the documention, I've added a test! (It passes in my fork branch, but GitHub won't let me run the workflow in this PR.)

I've also rebased from master. Hopefully this is all good to go now.

As a side note, if there's a way to run the tests locally then it would be great to have that added to the aREADME.

Thanks for the great extension.

willstott101 commented:
Great thanks. Sorry it took me a while to get back to this.
